### PR TITLE
bug: video background alt text appearing as 'A', 'B' etc.

### PR DESCRIPTION
**Pull request by AshKyd**

### CODE REVIEW

File: src/app/components/VideoPlayer/index.js [hunk 1]
```diff
@@ -106,12 +107,17 @@ const VideoPlayer = ({
   }
 
   if (!title) {
-    title = String.fromCharCode(getNextUntitledMediaCharCode());
```
Comment (AshKyd) [inline src/app/components/VideoPlayer/index.js:109]: I don't understand why we were doing this. This fromCharCode was returning "A", "B", "C" etc based on the video index.

This might be ok if we needed a unique player identifier, but it was being used for the alt text.

Can you think of what, if anything this will break?

**Comment by AshKyd** [comment]
Thought of a couple of edge cases last night. Let me revise.

**Comment by AshKyd** [comment]
Ok I think this is good. The change makes sure we only ever aria-hide the video if it's ambient and has no title.

**Comment by AshKyd** [comment]
Ok another change, while testing I found we were getting whitespace alt text `" "` that needed trimming. This looks good now

File: src/app/components/VideoPlayer/index.js [hunk 2]
```diff
@@ -105,13 +106,20 @@ const VideoPlayer = ({
...
+    title = '';
   }
 
   const placeholderEl = Sizer(ratios);
 
-  const videoEl = html`<video preload="none" tabindex="-1" aria-label="${title}"></video>`;
+  const videoEl = html`<video
+    preload="none"
+    tabindex="-1"
+    aria-label="${title || ''}"
```
Comment (AshKyd) [inline src/app/components/VideoPlayer/index.js]: We've already sanitised this above

```suggestion
    aria-label="${title}"
```

File: src/app/components/VideoPlayer/index.js [hunk 3]
```diff
@@ -231,6 +239,19 @@ const VideoPlayer = ({
...
+      this._alternativeText = sanitisedText;
+      const videoEl = this.getVideoEl?.();
+      if (!videoEl) {
+        return;
+      }
+      videoEl.setAttribute('aria-label', sanitisedText);
+      videoEl.setAttribute('aria-hidden', !sanitisedText ? 'true' : 'false');
+    },
+    get alternativeText() {
+      return this._alternativeText;
```
Comment (AshKyd) [inline src/app/components/VideoPlayer/index.js]: What if this was `return this.getVideoEl?.()?.getAttribute('aria-label') || ''`?

I think it will always be set and available, and we could remove the `_alternatieText` property

**Comment by AshKyd** [comment]
I tried to rebase and broke everything. Reopening in https://github.com/abcnews/odyssey/pull/83
